rdkit/Chem/SaltRemover.py

```diff
@@ -146,6 +195,38 @@ def StripMol(self, mol, dontRemoveEverything=False):
     2
 
     """
+    strippedMol = self._StripMol(mol, dontRemoveEverything)
+    return strippedMol.mol
+
+  def StripMolWithDeleted(self, mol, dontRemoveEverything=False):
+    """
+    Strips given molecule and returns it, with the fragments which have been deleted.
+
+    >>> remover = SaltRemover(defnData="[Cl,Br]")
+    >>> len(remover.salts)
+    1
+
+    >>> mol = Chem.MolFromSmiles('CN(C)C.Cl.Br')
+    >>> res, deleted = remover.StripMolWithDeleted(mol)
+    >>> Chem.MolToSmiles(res)
+    'CN(C)C'
+    >>> [Chem.MolToSmarts(m) for m in deleted]
+    ['[Cl,Br]']
+
+    >>> mol = Chem.MolFromSmiles('CN(C)C.Cl')
```

Comment:
I think the doctests are super helpful for understanding (and testing) what the code does.

Two minor suggestions:

1) An example showing what `StripMolWithDeleted()` returns when mol is `'CN(C)C.Cl.Cl'`.
2) An example demonstrating use of the namedtuple returned from `StripMolWithDeleted()` instead of doing the straightforward tuple assignment of the results.

These could be the same example, of course.